src/main/kotlin/pt/up/fe/ni/website/backend/model/Project.kt

```diff
@@ -7,5 +7,6 @@ class Project(
     title: String,
     description: String,
     var isArchived: Boolean = false,
+    technologies: List<String>,
```

Comment:
```suggestion
    val technologies: List<String>,
```
Without val this'd be a useless parameter without creating the respective field. Read more in https://www.baeldung.com/kotlin/constructors#primary-constructor

Reply:
I tried that before but got this error, therefore i decided to remove it 

<img width="555" alt="image" src="https://user-images.githubusercontent.com/87575749/214252444-9664541c-9341-4cef-b39c-48a5293b7874.png">

Reply:
Error disappeared 🤷‍♂️

Reply:
It's always a good idea to research errors/warning instead of trying to fix them blindly. Your IDE plugin is probably referring to a previous version of Hibernate ORM (the one who maps the models to the DB) that didn't support conversion of simple arrays and lists. Instead, you would have to define a separate table (e.g. `@ElementCollection`) or a converter (transform list to string and vice versa).

The ORM is now capable of treating simple lists as a "basic plural" and maps it to just one column, which seems appropriate for this case. Read "JDBC array type support" at https://in.relation.to/2022/06/24/hibernate-orm-61-features/